core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -24,6 +24,7 @@ import java.nio.ByteBuffer
 import java.util.concurrent.{ConcurrentHashMap, TimeUnit}
 
 import scala.collection.JavaConverters._
+import scala.collection.JavaConversions._
```

Comment:
`JavaConversions` is discouraged (its hard for the reader to see where the conversion is happening).  stick to `JavaConverters` and explicitly call `asScala` / `asJava`